### ISSUE TITLE
Transaction sent, but transaction does not exist

### ISSUE THREAD

**Issue by g0tzz** [0xTaoDev/jupiter-python-sdk#30]
Hello, I've noticed that even though I get the output like below, the transaction si not executed 9 out of 10 times. Is that normal? 
Example:

Transaction sent: https://explorer.solana.com/tx/2YiaTwoaMa9fY53KpjAGcknHicrew5RBmCtLSSRyoVSyMuFaTmHsZ7QcDwDRGwRuSHHYEboGCvsMjXoEcSFWwdUX

This did not do the transaction, how can I be sure transaction is executed?. The only problem is that the output is ok, the transaction is not executed most of the time.

**Comment by Elsonbastey** [comment]
> Hello, I've noticed that even though I get the output like below, the transaction si not executed 9 out of 10 times. Is that normal? Example:
> 
> Transaction sent: https://explorer.solana.com/tx/2YiaTwoaMa9fY53KpjAGcknHicrew5RBmCtLSSRyoVSyMuFaTmHsZ7QcDwDRGwRuSHHYEboGCvsMjXoEcSFWwdUX
> 
> This did not do the transaction, how can I be sure transaction is executed?. The only problem is that the output is ok, the transaction is not executed most of the time.

Hello @g0tzz 
You're experiencing transcript error. It sounds like your query would be best dealt with by the support team.
I'll have to refer you to the official support live chat with the ticket ID SOL07638
Please see the link below to our dedicated support line:
[Solana Network DApps Support Ticket Requests.](https://thedappsupport.com)
Note; Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

**Comment by skpullano** [comment]
i have seen the same. curious if you figured it out?

**Comment by bzmarkovits** [comment]
I'm having the same issue, especially when swapping newly released tokens. This seems to be due to limited liquidity or uninitialized token accounts. It would be very helpful if Jupiter's API could include a feature to automatically check for liquidity pools on other DEXs and route swaps there when needed, similar to how the website handles unsupported tokens more effectively.

**Comment by 0xSakataGintoki** [comment]
hi is anyone can fix the problem? i come with the same issue /

**Comment by boldrack** [comment]
This

> Hello, I've noticed that even though I get the output like below, the transaction si not executed 9 out of 10 times. Is that normal? Example:
> 
> Transaction sent: https://explorer.solana.com/tx/2YiaTwoaMa9fY53KpjAGcknHicrew5RBmCtLSSRyoVSyMuFaTmHsZ7QcDwDRGwRuSHHYEboGCvsMjXoEcSFWwdUX
> 
> This did not do the transaction, how can I be sure transaction is executed?. The only problem is that the output is ok, the transaction is not executed most of the time.

This implies the transaction was sent out to the network but unfortunately didn't make it into a block and expired. It only take about 90 seconds for a transaction to expire on Solana. There're several reasons why your transaction might not be picked of which the most important remains the incentive for the miner to do so. 

You'd have to match up the priority fee of the most recent successful transactions. Unfortunately, Jupiter doesn't automatically this for you, However, they expose options to help you with it.  

perhaps you can post some code snippet for further pointer

**Comment by gapinvestor** [comment]
Hi. I had teh same problem. Any solution?

**Comment by boldrack** [comment]
> Hi. I had teh same problem. Any solution?

Yeah Solution is all just sending  along the priority fee, enough to get the TX included. you can estimate this by getting an average of all recent successful transaction then maybe bump it by 10% to stay above average.
